src/server/server.go

```diff
@@ -63,6 +69,32 @@ func PprofMem(w http.ResponseWriter, r *http.Request) {
         }
 }
 
+func PprofCpuStart(w http.ResponseWriter, r *http.Request) {
+	log.Printf("Starting cpu profiling\n")
+	w.Header().Add("Content-Type", "application/octet-stream")
+
+	if err := pprof.StartCPUProfile(w); err != nil {
```

Comment:
Instead of writing the profile info back to w, I would write it to a temp file and just send back a confirmation it is started.  I think users will want the profiling data after they call stop, so the stop call should (1) stop the profiling (2) read the temp file and (3) send the profile data back.

Reply:
Yes, I think this makes more sense. I will use a temp file :)